custom/icds_reports/migrations/sql_templates/database_functions/insert_into_ccs_record_monthly.sql

```diff
@@ -6,17 +6,28 @@ DECLARE
   _tablename text;
   _ucr_ccs_record_table text;
   _ucr_pregnant_tasks_table text;
+  _agg_thr_form_table text;
```

Comment:
Considering how much faster it was, you may want to go ahead and move to the type of aggregation used here: https://github.com/dimagi/commcare-hq/pull/20957